acceptance/analyzer_test.go

```diff
@@ -112,6 +112,62 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 		})
 	})
 
+	when("Platform API = 0.6", func() {
```

Comment:
```suggestion
	when("Platform API > 0.5", func() {
```
(for consistency with other context)